sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -81,8 +82,21 @@ class SampleApplication : Application() {
         "127.0.0.1"
     )
 
+    private val tracingHeaderTypes = setOf(
+        TracingHeaderType.DATADOG,
+        TracingHeaderType.TRACECONTEXT,
+        TracingHeaderType.B3,
+        TracingHeaderType.B3MULTI
```

Comment:
do we have this set up on the backend side? is it useful?

Reply:
nope this was just for tests will be removed in the actual PR